preview/Win7Msix/Win7MSIXInstaller/MsixRequest.hpp

```diff
@@ -50,10 +54,13 @@ class MsixRequest
     /// Called by CreateAndShowUI 
     void SetUI(UI* ui);
 
+    void SetMsixResponse(MsixResponse* msixResponse);

```

Comment:
>SetMsixResponse [](start = 9, length = 15)

is this being called outside? by whom?  my thoughts are either 1. this is created by caller and passed in through MsixRequest::Make(), or 2. the MsixRequest::Make itself creates it like you're doing but then doesn't expose a way to set it

Reply:
It is not being called anywhere, I forgot to remove it, removed the same.